packages/spacecat-shared-rum-api-client/test/rum-api-client.test.js

```diff
@@ -205,4 +205,126 @@ describe('rum api client', () => {
     const rumApiClient = RUMAPIClient.createFrom({ env: { RUM_DOMAIN_KEY: 'hebele' } });
     await expect(rumApiClient.getDomainList()).to.eventually.eql(['spacecat.com', 'spacekatze.com']);
   });
+
+  it('returns data when getConversionData api is successful', async () => {
+    nock('https://helix-pages.anywhere.run/helix-services')
+      .get('/run-query@v3/rum-sources-targets')
+      .query({
+        domainkey: 'hebele',
+        interval: 7,
+        offset: 0,
+        limit: 101,
+        checkpoint: 'convert',
+        url: 'http://spacecat.com',
+      })
+      .reply(200, JSON.stringify({
+        results: {
+          data: [{
+            checkpoint: 'convert',
+            source: 'get-a-demo',
+            target: null,
+            ids: 56,
+            pages: 3,
+            topurl: 'https://www.bamboohr.com/g2/',
```

Comment:
I would replace all bamboohr reference, so we don t have any real customer url s in the tests, apart from that it looks good

Reply:
Done.